distributed/client.py

```diff
@@ -753,6 +753,21 @@ def sync(self, func, *args, asynchronous=None, callback_timeout=None, **kwargs):
                 self.loop, func, *args, callback_timeout=callback_timeout, **kwargs
             )
 
+    def __repr__(self):
```

Comment:
You have two __repr__ in client class now, is this voluntary? This looks like a copy paste mistake.